flow/types.js

```diff
@@ -60,4 +60,32 @@ declare module 'icarus-backend' {
     status: number,
     data: any
   }
+
+  // either txInput or txOutput
+  declare type Movement = {
+    txid: number,
+    address: string,
+    value: number,
+  }
+
+  declare type MovementEntry = Array<any>
+
+  declare type CoinObject = {
+    getCoin: Big
+  }
+
+  declare type Tx = {
+    id: number,
```

Comment:
is the id really a number and do we really need it? I mean `getDistinctTxInputs` function expects an array of strings, but iif this is true, it's receiving actually an array of numbers in the place it is used (line 108 in legacy-routes.js)

But can't we use the hashes alone? If it's inefficient, wouldn't creating an index for it fix it? Exposing the id from the db kind of smells to me as it's a low level thing.